fastkml/kml.py

```
@@ -244,6 +246,53 @@
+            tree = config.etree.tostring(
+                    element,
+                    encoding="UTF-8",
+                    xml_declaration=xml_declaration
+                ).decode(
+                    "UTF-8",
+                )
+
+        if file_path.suffix == ".kmz":
+            with zipfile.ZipFile(file_path, 'w', zipfile.ZIP_DEFLATED) as kmz:
+                kmz.write(file_path, arcname=os.path.basename(file_path))
+        else:
```

Ruff

`os.path.basename()` should be replaced by `Path.name`

(PTH119)